docs/docker/docker-troubleshooting.md

```diff
@@ -134,5 +134,14 @@ Start docker-sync so your docker-sync containers and volumes will be recreated:
 docker-sync start
 ```
 
+## Application is slow on Mac
+We focus on enhancing performance of application on all platforms.
+With Docker for Mac and Docker for Windows there are known performance issues because all project files need to be synchronized from host computer to application running in virtual machine.
```

Comment:
There are known some issues with docker on mac and docker on windows, because of a need of synchronization of all project files form host ...